Makefile

```diff
@@ -5,68 +5,60 @@ GOOS=$(shell go env GOOS)
 GOARCH=$(shell go env GOARCH)
 
 ifeq ($(GOOS),linux)
-MD5=md5sum
+  MD5SUM=md5sum
 else
-MD5=md5 -q
+  MD5SUM=md5 -q
 endif
 
 help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-18s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
-build: fmt vet ## Build dtm & plugins locally.
+build: build-core build-plugins md5 ## Build dtm core & plugins locally.
+	rm -f dtm
+	mv dtm-${GOOS}-${GOARCH} dtm
```

Comment:
why rename it？ such dtm packages would conflict if released.
Or, run an other script before release?

Reply:
It's easier to use at gh actions.